Dockerfile

```
@@ -2,15 +2,30 @@
 # Use of this source code is governed by a BSD-style
 # license that can be found in the LICENSE file.
 
-FROM golang:1.13.4-alpine AS builder
-
+FROM golang:1.13.4 AS tester
```

```suggestion
FROM golang:1.13.5 AS tester
```

it took me so long to review your PR that there's a new version of Go, sorry about that 🙊

---

No worries. I rebased from master, where you also updated